### PR TITLE
add rubocop to db folder

### PR DESCRIPTION
**Pull request by pandermatt**
I think its important to also have some linters in the migrations. We introduced linters to migration on https://github.com/renuo/loyalty and Edge and I liked it. Since it can still detect some in-reversible migrations or code smells

### CODE REVIEW

File: templates/.rubocop.yml [hunk 1]
```diff
@@ -36,11 +36,28 @@ Style/HashTransformValues:
 Style/HashAsLastArrayItem:
   Enabled: false
 
+Metrics/MethodLength:
+  CountAsOne:
+    - array
+    - hash
+    - heredoc
+  Exclude:
+    - 'db/**/*'
```
Comment (pandermatt) [inline templates/.rubocop.yml:45]: as suggested in https://github.com/renuo/applications-setup-guide/pull/268
I would also like to add CountAsOne to MethodLength and not only to ClassLength

File: templates/.rubocop.yml [hunk 2]
```diff
@@ -18,8 +18,8 @@ Style/Documentation:
 
 Style/MixinUsage:
   Exclude:
-  - 'bin/setup'
-  - 'bin/update'
+    - 'bin/setup'
+    - 'bin/update'
```
Comment (pandermatt) [inline templates/.rubocop.yml:22]: I also changed the spacing to be consistent with the rest of the file

**Review by coorasse — APPROVED** [review]
agree :)

**Review by Liberatys — APPROVED** [review]
👍🏻 Noice

**Review by schmijos — APPROVED** [review]

**Pull request merged**